spec/service_actor/arguments_validator_spec.rb

```diff
@@ -40,4 +40,47 @@
         )
     end
   end
+
+  describe ".validate_default_value" do
+    include_context "with mocked `Kernel.warn` method"
+
+    [[], {}, {a: 1}, {a: []}.freeze, Struct.new(:a).new].each do |mutable_value|
+      context "when `Ractor` API is supported" do
+        include_context "skip on MRI 2.7"
```

Comment:
It’s probably time to stop supporting MRI 2.7 though, since even 3.0.7 isn’t maintained anymore. We can remove 2.7 from the matrix if you prefer or keep this for the time being.

Reply:
Is that okay for you let's do it! Here's a pull request: https://github.com/sunny/actor/pull/194